app/models/agent.rb

```diff
@@ -26,6 +26,9 @@ class Agent < ApplicationRecord
   scope :active, -> { where(deleted_at: nil) }
   scope :order_by_last_name, -> { order(Arel.sql('LOWER(last_name)')) }
   scope :secretariat, -> { joins(:service).where(services: { name: 'Secrétariat'.freeze }) }
+  scope :available_for_rdv, lambda { |motif, service|
```

Comment:
also in my mind `available_for_rdv` refers to being available on a specific time for a specific rdv, whereas here it's a more generic , someone can do that motif.

I'd be happier with something like `can_perform_motif`, wdyt?